scripts/quests/sandoria/A_Vicasques_Sermon.lua

```diff
@@ -0,0 +1,140 @@
+-----------------------------------
+-- A Vicasques Sermon
+-- !addquest 0 9
+-- NPC: Abioleget !pos 128.771 0.000 118.538 231
+-- NPC: Andelain !pos 664.231 -12.849 -539.413 101
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/items')
+require('scripts/globals/zone')
+-----------------------------------
+local ID = require("scripts/zones/Northern_San_dOria/IDs")
+local ronfaureID = require("scripts/zones/East_Ronfaure/IDs")
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.THE_VICASQUE_S_SERMON)
+
+quest.reward =
+{
+    fame = 30,
+    fameArea = xi.quest.fame_area.SANDORIA,
+    item = xi.items.BRASS_RING,
+    title = xi.title.THE_BENEVOLENT_ONE,
+}
+
+quest.sections =
+{
+    -- Speak to Abioleget after completing Waters of the Cheval
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.WATERS_OF_THE_CHEVAL) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Abioleget'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(589)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [589] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    -- Trade Blue peas to Andelain in East Ronfure
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.NORTHERN_SAN_DORIA] =
+        {
+            ['Abioleget'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(600)
+                    elseif quest:getVar(player, 'Prog') == 0 then
+                        player:showText(npc, ID.text.ABIOLEGET_PEAS, xi.items.BLUE_PEAS, 70)
+                    end
+                end,
+
+                onTrade = function(player, npc, trade)
+                        local gil = trade:getGil()
+                        local count = trade:getItemCount()
+
+                        if gil == 70 and count == 1 then
```

Comment:
can players do this infinitely? or do we need a var to prevent them from doing it more than once?

Reply:
They are more expensive here than NPC, and should only be available if the quest is active.